build.gradle

```diff
@@ -100,7 +100,6 @@ spotless {
 	// 	indentWithSpaces(2)  // YAML cannot contain tabs: https://yaml.org/faq.html
 	// }
 	java {
-		removeUnusedImports()
```

Comment:
can you post a screenshot? I've never seen that. I did test this and confirm that unused imports were no longer being removed, once we remove the line in question.

Reply:
I returned it, it works fine after initialization, but i had a problem when initializing. If i encouter it again, i'll open a separate issue. It might be the problem with commons actually...